sharpd/sharp_zebra.c

```diff
@@ -989,6 +989,41 @@ static int sharp_zebra_process_srv6_locator_chunk(ZAPI_CALLBACK_ARGS)
 	return 0;
 }
 
+static int sharp_zebra_process_neigh(ZAPI_CALLBACK_ARGS)
```

Comment:
This function always return 0.
What is the difference between failure and success?

Reply:
This is a bit of a historical baggage that is carried around at the moment, that needs to be cleaned up.  The original code all had this idea of returning error messages.  But whenever a non-zero was returned it was always ignored.  When the current handler code was abstracted, by Quentin, to use arrays of function pointers it got carried over.  The original coder of a lot of functionality in FRR had this idea of returning 0 or 1 for success and immediately throwing the answer away and never using it.  We've cleaned up large swaths.  This is just another example of something that needs to be fixed from my perspective.